### PR TITLE
Create makefile for JDK22 version

### PR DESCRIPTION
**Pull request by KapilPowar**
[/runtimes/backlog/issues/1267](https://github.ibm.com/runtimes/backlog/issues/1267)

-   Create .mk file for JDK22 version
-   Update .mk file for JDK8 and JDK21 version

CC: @llxia

### CODE REVIEW

**Comment by KapilPowar** [comment]
Grinder Tests
JDK 8 -- Grinder 1958
JDK 21 -- Grinder 1959
JDK 22 -- Grinder 1955, Grinder 1960

**Review by llxia — APPROVED** [review]
LGTM

File: jck/subdirs/jck22.mk [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
.../stmt431;lang/STMT/stmt432;lang/STMT/stmt433;lang/STMT/stmt434;lang/STMT/stmt435;lang/STMT/stmt436;lang/STMT/stmt437;lang/STMT/stmt438;lang/STMT/stmt439;lang/STMT/stmt440;lang/STMT/stmt441;lang/STMT/stmt442;lang/STMT/stmt443;lang/STMT/stmt444;lang/STMT/stmt445;lang/STMT/stmt446;lang/STMT/stmt447;lang/STMT/stmt448;lang/STMT/stmt450;lang/STMT/stmt451;lang/STMT/stmt452;lang/STMT/stmt453;lang/STMT/stmt454;lang/STMT/stmt455;lang/STMT/stmt456;lang/STMT/stmt457;lang/STMT/stmt458;lang/STMT/stmt459;lang/STMT/stmt460;lang/STMT/stmt461;lang/STMT/stmt462;lang/STMT/stmt463;lang/STMT/stmt464;lang/STMT/stmt465;lang/STMT/stmt466;lang/STMT/stmt467;lang/STMT/stmt468;lang/STMT/stmt469;lang/STMT/stmt470;lang/STMT/stmt471;lang/STMT/stmt472;lang/STMT/stmt473;lang/STMT/stmt474;lang/STMT/stmt475;lang/STMT/stmt476;lang/STMT/stmt482;lang/STMT/stmt483;lang/STMT/stmt484;lang/STMT/stmt485;lang/STMT/stmt486;lang/STMT/stmt487;lang/STMT/stmt488;lang/STMT/stmt489;lang/STMT/stmt490;lang/STMT/stmt491;lang/STMT/stmt492;lang/STMT/stmt493;lang/STMT/stmt494;lang/STMT/stmt495;lang/STMT/stmt496;lang/STMT/stmt497;lang/STMT/stmt498;lang/STMT/stmt499;lang/STMT/stmt500;lang/STMT/stmt501;lang/STMT/stmt502;lang/STMT/stmt503$(Q)
```
Comment (JasonFengJ9) [inline jck/subdirs/jck22.mk:75]: Please add a new line, note the red circle at the end of L75

**Review by smlambert — APPROVED** [review]
👍

**Pull request merged**